meta-resin-common/recipes-support/resinhup/resinhup/run-resinhup.sh

```diff
@@ -156,10 +156,10 @@ function runPostHacks {
             dockerpid=$(pidof rce)
             kill -9 $dockerpid &> /dev/null
 
-            if [ -d "$DATA_MOUNTPOINT/docker" ]; then
```

Comment:
I think that this change from BTRFS_MOUNTPOINT to DATA_MOUNTPOINT  makes sense as in the future we will switch this anyway.